src/site/content/en/lighthouse-performance/lighthouse-largest-contentful-paint/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Largest Contentful Paint
+description: |
+  Learn about Lighthouse's Largest Contentful Paint metric and
+  how to measure and optimize it.
+date: 2019-10-09
+web_lighthouse:
+  - largest-contentful-paint
+---
+
+<!-- TODO https://web.dev/largest-contentful-paint already exists.
+     What should the URL for this page be? -->
+
+Largest Contentful Paint (LCP) is one of the metrics
+tracked in the **Performance** section of the Lighthouse report.
+Each metric captures some aspect of page load speed.
+
+<!-- TODO update this depending on how the report displays LCP. Milliseconds? -->
+
+Lighthouse displays LCP in milliseconds:
+
+<!-- TODO update this once LCP lands in the report UI. -->
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="largest-contentful-paint.png"
+       alt="A screenshot of the Lighthouse Largest Contentful Paint audit">
+</figure>
+
+## What LCP measures
+
+LCP measures when the largest content element visible in the viewport is
```

Comment:
I don't have any good alternatives, but `largest content element visible in the viewport` is confusing (`visible` makes me need to parse the sentence carefully)

Reply:
Thanks, I think I can just remove `visible`